test/integration/guest_user_test.rb

```diff
@@ -150,4 +150,47 @@ class GuestUserTest < ActionDispatch::IntegrationTest
 
     refute page.has_content?("Order #{order.id}")
   end
+
+  test "an unauthorized user can add projects to pending_loans and see the
+  projects on the pending_loans show page" do
+    tenant = create(:tenant)
+    tenant.projects << create(:project, price: 8900)
+
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.